src/api/spec/helpers/webui/webui_helper_spec.rb

```diff
@@ -297,14 +164,6 @@
     end
   end
 
-  describe '#repo_status_icon' do
-    skip('Please add some tests')
-  end
-
-  describe '#tab' do
-    skip('Please add some tests')
-  end
-
   describe '#render_dialog' do
```

Comment:
nitpick: I think the ```#render_dialog``` can be removed as well, at least I couldn't find something in the webui_helper that seemed to be related

Reply:
Added. Thanks!